.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: python
 jobs:
   include:
-    - python: 3.5
-      env: TOXENV=py35
-    - python: 3.6
-      env: TOXENV=py36
     - python: 3.7
       env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
+    - python: 3.9
+      env: TOXENV=py39
+    - python: 3.10
+      env: TOXENV=py310
```

Comment:
Let's drop support for 3.10 until Travis is ready.

Reply:
ok

Reply:
ok